README.md

```diff
@@ -1,2 +1,38 @@
 # solidity-cborutils
 Utilities for encoding [CBOR](http://cbor.io/) data in solidity
+
+## Install
+
+```bash
+$ git clone https://github.com/smartcontractkit/solidity-cborutils.git
+$ cd solidity-cborutils
+$ yarn install
```

Comment:
npm install?

Reply:
`npm` works for me. I'll get that updated.